doc/userguide/rules/payload-keywords.rst

```diff
@@ -691,11 +693,25 @@ the reassembled stream.
 The checksums will be recalculated by Suricata and changed after the
 replace keyword is being used.
 
+.. _pkt-data:
+
+pkt_data
+--------
+
+``pkt_data`` is a sticky buffer that allows for payload inspection from the
+beginning of the normalized packet data. As a sticky buffer, ``pkt_data``
+requires the usage of the :ref:`content` keyword for the buffer inspection.
```

Comment:
it requires a content inspection keyword, not just `content`. Also things like pcre, byte_test, etc.

Reply:
🙇🏽